views/availabilities.twig

```diff
@@ -1,12 +1,15 @@
-<html>
-  <head>
+{% extends "master.twig" %}
```

Comment:
Really like these changes, so much cleaner. I'd rename the master.twig to base.twig though, as that is a more expected name for that kind of template.